dedupe_activedirectory_users/dedupe_by_displayname.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
```

Comment:
verify that `-e` is set so we will exit on failed commands
```suggestion
set -e
```